### PR TITLE
[TA] SDK now targets 3.1-preview.2

### LINKED ISSUE #14564: Update code to 3.1-preview.2

### PR DESCRIPTION
**Pull request by maririos**
New features generated + CI enabled with a PPE endpoint (will delete once service deploys to other regions) + recordings.
No new functionalities or public API changed with this PR

Fixes: https://github.com/Azure/azure-sdk-for-net/issues/14564

### CODE REVIEW

File: sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## 5.1.0-beta.1 (Unreleased)
 ### Breaking changes
-- It defaults to the latest supported API version, which currently is `3.1-preview.1`.
+- It defaults to the latest supported API version, which currently is `3.1-preview.2`.
```
Comment (mssfang) [inline sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md:5]: nit: Do we want to consider upgrading to a newer version as a new feature change?

Comment (maririos) [reply sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md:5]: Newer version of what? Not sure I follow

Comment (maririos) [reply sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md]: Moved to `New Features`

**Review by iscai-msft — APPROVED** [review]

**Pull request merged**